posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,343 @@
+---
+layout: post
+title: "Jakarta EE 10 Core Profile debuts in Open Liberty 22.0.0.9-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/reece-nana
+author_github: https://github.com/reece-nana
+seo-title: Jakarta EE 10 Core Profile debuts in Open Liberty 22.0.0.9-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.9-beta brings you the hot-off-the-press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release to update the Jakarta EE specifications since Java EE 8 in 2017.  MicroProfile OpenAPI 3.1 and Password Utilities 1.1 are also available in this beta release.
+blog_description: Open Liberty 22.0.0.9-beta brings you the hot-off-the-press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release to update the Jakarta EE specifications since Java EE 8 in 2017.  MicroProfile OpenAPI 3.1 and Password Utilities 1.1 are also available in this beta release
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta
+Reece Nana <https://github.com/reece-nana>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.9-beta brings you the hot-off-the-press link:https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE 10 Core Profile]. Jakarta EE 10 is the first release to update the Jakarta EE specifications since Java EE 8 in 2017. While the Jakarta EE 10 release is still around the corner, Open Liberty has consumed many of the Jakarta EE 10 specifications and put them in the 22.0.0.9-beta.  The features that make up the new Jakarta EE 10 Core Profile are available in this beta release. 
```

Comment:
There was supposed to be an update in this blog post for the Jakarta Concurrency 3.0 feature (concurrent-3.0) as documented here:  https://github.com/OpenLiberty/open-liberty/issues/20014#issuecomment-1190373788.  @njr-11 did we decide not to do it?

Reply:
The issue was tagged 22009-beta so it was supposed to be part of this beta blog as we had discussed in the Jakarta scrum.

Reply:
I think that's a good catch; I'm going to publish the blog as-is, and then make the update to add Concurrency 3.0.